src/components/PageDefinitions.js

```diff
@@ -39,7 +40,7 @@ export class PageDefinitions extends AbstractPageDefinitions {
         const definitionSpec = pako.inflate(base64js.toByteArray(path), { to: 'string' })
         this.loadFromListSpec(JSON.parse(definitionSpec))
       } catch (e) {
-        dispatch(uiNotificationNew({ type: 'warning', message: 'Loading components from URL failed', timeout: 5000 }))
+        uiWarning(dispatch, 'Loading components from URL failed')
```

Comment:
Using the above mentioned Redux method, the use of `dispatch` results not needed anymore